fontbe/src/fvar.rs

```diff
@@ -32,6 +32,9 @@ fn generate_fvar(static_metadata: &StaticMetadata) -> Option<Fvar> {
     let reverse_names: HashMap<_, _> = static_metadata
         .names
         .iter()
+        // To match fontmake we should use the font-specific name range and not reuse
```

Comment:
it's not just to match fontmake, but to match the spec (search for "255") https://learn.microsoft.com/en-us/typography/opentype/spec/fvar